src/mdx-components.tsx

```
@@ -42,6 +42,9 @@
 import { UsageBasedPricingDemo } from '@/components/usage-based-pricing-demo';
 import { DetailedUsageTableDemo } from '@/components/detailed-usage-table-demo';
 import { PricingTableSevenDemo } from '@/components/pricing-table-seven-demo';
+import { PricingTableEightDemo } from '@/components/pricing-table-eight-demo';
+import { PaymentCardDemo } from '@/components/payment-card-demo';
+import { PlanRecommendationWidgetDemo } from '@/components/plan-recommendation-widget-demo';
 import {PaymentCardDemo} from '@/components/payment-card-demo';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'PaymentCardDemo'. Consider to delete it or rename it.

'PaymentCardDemo' is defined here:


(lint/suspicious/noRedeclare)